e2e/cypress/integration/emoji/wrap_emoji_spec.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Wrap emojis', () => {
+    before(() => {
+        cy.apiLogin('user-1');
+        cy.visit('/');
+        cy.clearLocalStorage();
+    });
+
+    it('MM-18712 Large number of emoji reactions wrap into multiple lines', async () => {
+        //cy.get('#post_textbox').type(':cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: {enter}')
+        cy.postMessage(':dog: :dog: :dog: :dog: :dog: :dog: :dog: :dog: :dog: :dog: :dog: :dog: :dog: :dog: :dog: :dog: :dog: :dog: :dog: :dog: :dog: :dog: :dog: :dog: :dog:');
```

Comment:
I realized that the test requirement is a bit confusing (sorry for that).  It's not posting emojis but it's actually adding number of reactions to a post.

Reply:
Will address this weekend